linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_audit/tests/common.sh

```diff
@@ -1,3 +1,4 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle
```

Comment:
I think we don't need to add this here, since you added to the tests and you source this file from the tests.